package.json

```diff
@@ -8,12 +8,6 @@
     "./src/app/legacy/psammead/psammead-locales/**/*.*",
     "./cypress/**/*.*"
   ],
-  "resolutions": {
-    "@optimizely/js-sdk-utils": "0.4.0",
-    "winston@3.8.2": "patch:winston@npm:3.8.2#.yarn/patches/winston-npm-3.8.2-2035e9cac4.patch",
```

Comment:
Does this mean that our winston patch is no longer applied? I believe we might still need to keep this because memory leak? (cc @andrewscfc)

Reply:
This was the original PR: https://github.com/bbc/simorgh/pull/10402

Side note: should we consider replacing winston because of this? It's been on my 10% list, but I have not gotten around to it yet. We also have to ignore any upgrades to winston in dependabot (and ideally we would have a ticket in the backlog - either to try and push the winston maintainers for a fix, or to replace it with something that we can continue to upgrade)

Reply:
Gotcha - so we're OK to remove it from resolutions. (But probably do need to figure out what to do with `winston` in the future).

Reply:
Could look to replace it or try updating it to the latest version once everything is moved to Lambda. Given the dynamic nature of Lambdas spinning up/down, the problem of logs not clearing from memory by Winston may just be solved.

Very hard to say for sure though.

Reply:
Ooooh very true. That would be great if that was the case. I've added a ticket for replacing winston - https://jira.dev.bbc.co.uk/browse/WORLDSERVICE-914 - but it could be updated to be: upgrade it to latest version?

Reply:
I created a second ticket for upgrading winston: https://jira.dev.bbc.co.uk/browse/WORLDSERVICE-918